opencog/cython/opencog/type_constructors.pyx

```diff
@@ -0,0 +1,132 @@
+
+# DO NOT EDIT THIS FILE! This file was automatically generated from atom
+# definitions in types.script by the macro OPENCOG_ADD_ATOM_TYPES
```

Comment:
In the Scheme syntax, the atom is defined as a list that may contain a TruthValue object, for example,
```(ConceptNode "Anna" (stv 0.1667 1))``` as in
https://github.com/opencog/opencog/blob/master/opencog/python/pln/examples/tuffy/smokes/smokes.scm#L17
but perhaps something like the syntax you have illustrated makes more sense for Python. Or maybe ```ConceptNode("dog").tv = SimplePLNTruthValue(0.5, 0.5)```. But what if you want to include the truth value at the same time as you are creating the atom?